src/website/src/pages/ec/components/inpage-navigation/index.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I was expecting to find the page under "Navigation" on the website (there's already a blank page here)